basyx.aasrepository/basyx.aasrepository-feature-registry-integration/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/feature/registry/integration/TestAasDescriptorFactory.java

```diff
@@ -0,0 +1,41 @@
+package org.eclipse.digitaltwin.basyx.aasrepository.feature.registry.integration;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class TestAasDescriptorFactory {
+
+	private static final String AAS_REPO_URL = "http://localhost:8081";
+
+	private static final String AAS_REPOSITORY_PATH_WITH_SLASH = "/shells";
+	private static final String AAS_REPOSITORY_PATH_WITHOUT_SLASH = "shells";
```

Comment:
Either stick to building the withSlash/withoutSlash for both path and repo_url in the test or define them consistenly as final vars; but don't mix both approaches.